catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/IcebergCatalogOperations.java

```diff
@@ -246,8 +246,14 @@ public IcebergSchema alterSchema(NameIdentifier ident, SchemaChange... changes)
       List<String> removals = new ArrayList<>();
       Map<String, String> updates = new HashMap<>();
       Map<String, String> resultProperties = new HashMap<>(metadata);
+      boolean commentUpdated = false;
```

Comment:
can we remove `commentUpdated`?  we could use `comment == null` to justify whether comment is updated